cmd/restic/cmd_backup.go

```diff
@@ -128,6 +122,8 @@ func init() {
 	f.MarkDeprecated("hostname", "use --host")
 
 	f.StringArrayVar(&backupOptions.FilesFrom, "files-from", nil, "read the files to backup from `file` (can be combined with file args/can be specified multiple times)")
+	f.StringArrayVar(&backupOptions.FilesFromVerbatim, "files-from-verbatim", nil, "read the files to backup from `file` (can be combined with file args/can be specified multiple times)")
+	f.StringArrayVar(&backupOptions.FilesFromRaw, "files-from-raw", nil, "read the files to backup from `file` (can be combined with file args/can be specified multiple times)")
```

Comment:
Can we take the opportunity to change the `/` to `, ` inside the parenthesis for these two lines and the one above them?

Reply:
Made it a semicolon.